src/components/Table/nestedCards.tsx

```diff
@@ -107,27 +112,15 @@ export function makeOpenOrCloseCard(openCards: NestedCardStyle[], kind: "open" |
  * For the first or last cell of actual content, wrap them in divs that re-create the
  * outer cards' padding + background.
  */
-export function maybeAddCardPadding(
-  columns: GridColumn<any>[],
-  openCards: NestedCardStyle[],
-  columnIndex: number,
-  div: any,
-): any {
-  const isFirst = columnIndex === 0;
-  const isFinal = columnIndex === columns.length - 1;
-  if (!isFirst && !isFinal) {
-    // Even if we don't need the nested color+padding of each open card, at
-    // least add the background color of the closed open card.
-    const card = openCards[openCards.length - 1];
-    return !card ? div : <div css={Css.bgColor(card.bgColor).$}>{div}</div>;
-  }
+export function maybeAddCardPadding(openCards: NestedCardStyle[], column: "first" | "final"): any {
+  let div: any = <div>&nbsp;</div>;
```

Comment:
Curious as to why you had to add this content? Was it to give it a height?

Reply:
Right, otherwise the inner divs didn't render.